### ISSUE TITLE
Impaired auditory rhythm perception

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#11403]
**Preferred term label:**
Impaired auditory rhythm perception

**Synonyms**
Impaired ability to perceive rhythm

**Definition (free text, please give PubMed ID)**
Impaired ability to perceive, discriminate, or predict temporal regularities and rhythmic patterns in auditory stimuli (such as musical rhythm/beat, tempo, or timing structure), despite intact peripheral hearing and preserved recognition of sounds, speech, and their semantic meaning (PMID:35710621; 32244259).

**Comment**
Individuals with this phenotype can recognize sounds and speech as meaningful but have difficulty detecting, differentiating, or synchronizing with rhythmic or temporal patterns within auditory sequences.

**Parent term (use hpo.jax.org/app)**
Disturbed sensory perception HP:0010524

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
@jagcsun I think this is not in the scope of the HPO since this is a trait that is not disease-related (as far as I know, but please reopen if there are disease associations!)
